blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "AKS Long Term Support: 24-Month Support Now Available for Every Kubernetes Version"
+description: "Every supported AKS Kubernetes version is now eligible for 24-month LTS support when you enable the Premium tier"
+date: 2025-07-25
+author: Kaarthis
+categories: general
+---
+
+In London at KubeCon EU 2025, AKS announced our expansion of what AKS Long Term Support (LTS) includes. Today, we're sharing more details about this offering that addresses one of the most critical challenges enterprises face when running Kubernetes at scale.
+
+From our conversations with customers, we consistently hear the same concerns: *"How do I balance Kubernetes innovation with the stability my business-critical applications require?"* and *"Why do I need to upgrade my clusters so frequently when my applications are running perfectly?"* On the flip side, customers also ask: *"If I don't upgrade frequently, how do I still ensure that I'm getting security fixes and ecosystem compatibility updates for my Kubernetes infrastructure?"* AKS LTS directly addresses these real-world challenges.
```

Comment:
If you add newlines after 400 characters, you won't trip the linter for these long lines.

Reply:
Done